ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -91,9 +91,9 @@ conformance_test(
     ],
 ) if not is_windows else None
 
+# some of the default and optional conformance tests run with lower concurrency
 conformance_test(
-    name = "conformance-test-contract-keys",
```

Comment:
I would leave `conformance-test-contract-keys` as is. We can add a fourth `conformance-test-extra` once we are reading to run the `ContractIdIT:RejectNonSuffixedCid,ContractIdIT:AcceptSuffixedCid` tests.

Reply:
I reverted the changes and add one `conformance-test-contract-ids`